main.js

```diff
@@ -0,0 +1,109 @@
+let level = 0
+let cleanTimer
+const colors = ['#ee3e3e', '#3e52f0', '#ef3ef0', '#f0ed3e', '#1e945f']
+const width = $('.main').width() 
+const height = $('.main').height() 
+
+const generateFrog = function () {
+  const $frog = $(`<div class='frog'><i class="fas fa-virus"></i></div>`)
+  const randWidth = Math.floor(Math.random() * width)
+  const randHeigth = Math.floor(Math.random() * height)
+  const randColor = Math.floor(Math.random() * colors.length)
+  const size = Math.max(20, randHeigth * 100 / height)
+
+  $('.main').append($frog)
+
+  $frog.css('font-size', `${size}px`)
+  $frog.css("top", `${randHeigth - size}px`)
+  $frog.css("left", `${Math.max(randWidth - 82, 82)}px`)
+  $frog.css('color', `${colors[randColor]}`)
+
+  return $frog
+}
+
+const timer = function () {
+  let sec = level 
+  const cc = () => {
```

Comment:
You should use clear and understandable names for classes, functions, and variables (for example: firstName etc.) and not something general like "arr", "str" etc.